CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 0.10.1 - 2017-04-11
+
+- Update fury to 3.0.0-beta.1
```

Comment:
Oh yeah, I forgot that fury is a dev dependency here and not a normal dependency.